assets/sass/protocol/includes/_mixins.scss

```diff
@@ -30,69 +28,53 @@
 // Mixins for CSS3 properties that still need prefixes in some browsers.
 @mixin transition($transition...) {
     transition: $transition;
-    transition: $transition;
 }
 
 @mixin transform($transform...) {
     transform: $transform;
-    transform: $transform;
 }
```

Comment:
Comment (non-blocking): it occurs to me that if we're removing vendor prefixes, then these mixins are all essentially now redundant? Not something we need to remove here and now, but we should probably file a follow up issue to remove them (which would also be a breaking change for Protocol).

Reply:
Yeah, I'll file an issue and we can talk over what this looks like a little more.